.github/workflows/appsignals-e2e-ec2-test.yml

```diff
@@ -58,6 +57,30 @@ jobs:
           role-to-assume: ${{ secrets.E2E_TEST_ROLE_ARN }}
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
 
+      - uses: actions/download-artifact@v3
+        if: ${{ inputs.caller-workflow-name }} == 'main-build'
+        with:
+          name: aws-opentelemetry-agent.jar
+
+      - name: Upload main-build adot.jar to s3
+        if: ${{ inputs.caller-workflow-name }} == 'main-build'
+        run: aws s3 cp ./aws-opentelemetry-agent.jar/aws-opentelemetry-agent-0.1.0-SNAPSHOT.jar s3://main-build-adot-staging-jar/aws-opentelemetry-agent.jar
```

Comment:
Is this correct? The `aws-opentelemetry-agent-0.1.0-SNAPSHOT.jar` is within the `aws-opentelemetry-agent.jar`?

Reply:
It was when I was downloading all the artifacts, but I changed it to only download the aws-opentelmetry-agent.jar, so not true anymore. 

Updated the code and also made the `0.1.0` into `*`